src/components/common/Modal/index.tsx

```diff
@@ -7,15 +7,42 @@ interface ModalProps {
   open: boolean;
   onClose: () => void;
   children?: ReactNode;
+  // Anchor content to the bottom of the screen. Primarily for mobile modals.
```

Comment:
I think you can change this to a doc comment

```suggestion
  /** Anchor content to the bottom of the screen. Primarily for mobile modals. */
```

so in VS Code it will show this description in autocomplete